Katana/Plastic/Value.swift

```diff
@@ -145,3 +187,65 @@ public struct Value: Equatable {
     return lhs.scalable == rhs.scalable && lhs.fixed == rhs.fixed
   }
 }
+
+/**
+ This allows `Value` instances to be initialized by integer and floating point literals,
+ having the scalable part equal to the numeric literal, and the fixed part equal to 0.
+ */
+extension Value: ExpressibleByIntegerLiteral, ExpressibleByFloatLiteral {
```

Comment:
In my opinion it's better to have `Value` conforming to `ExpressibleByIntegerLiteral` and `ExpressibleByFloatLiteral`, since they exist for this purpose: they make it clear to users of `Value` why using integer/float literals in operations expecting `Value`s work.

Defining an overload of the `+` operator for Anchor/Int and Anchor/Double would be more confusing IMHO.